packages/fxa-settings/config/webpack.config.js

```diff
@@ -305,6 +305,30 @@ module.exports = function (webpackEnv) {
         // This is only used in production mode
         new CssMinimizerPlugin(),
       ],
+     splitChunks: isEnvProduction ? {
```

Comment:
Note, if I left out the `isEnvProduction ?` part, this would cause the dev environment to hang. However, I can validate the bundle is created successfully when running `BUILD_TARGETS=prod yarn workspace fxa-settings build-react-prod`